markdown/get-started-basics/get-started-basics.md

```diff
@@ -0,0 +1,394 @@
+author: Aaron Lee
+summary: Absolute basics of first connecting to a Solace event broker
+id: get-started-basics
+tags: howto
+categories:
+environments: Web
+status: Published
+feedback link: https://solace.community/
+analytics account: UA-3921398-10
+
+# Absolute Solace Basics - Getting Started with Solace PubSub+ Event Broker
+
+
+
+
+
+
+## What you'll learn: Overview
+
+Duration: 0:01:00
+
+In this introductory CodeLab, you'll learn how to:
+
+* Setup a **Solace PubSub+ event broker**, and/or start a **PubSub+ Cloud** service
+* Login to the PubSub+ Manager GUI
+* Utilize test WebSockets applications to send and receive data using **Publish-Subscribe**, the fundamental communication pattern for event-driven applications
+* Improve routing and filtering using **topic hierarchies and wildcards**
+* Create a durable queue for **Guaranteed** Messaging
+* Test Guaranteed messaging to ensure no message loss
+* Enable **Replay** on the Solace broker, and test it
+
+![picture](img/broker-img.png)
+
+
+Positive
+: This CodeLab does not cover the Event Portal. That will be addressed in a separate CodeLab.
+
+
+
+
+
+
+
+## Prerequisites
+
+Duration: 0:01:00
+
+In order to complete this CodeLab, you will need to get access to a Solace PubSub+ event broker.
+
+At minimum, you will need to have internet access and a browser to use Solace PubSub+ Cloud.
+
+To run the software broker, you'll need to have Docker, VirtualBox, VMWare Player, or another similar technology available to you.  More details in the next step.
+
+
+
+
+
+
+
+## Starting a Solace PubSub+ Event Broker
+
+Duration: 0:10:00
+
+Let's get access to a Solace PubSub+ event broker!
+
+### Software Broker
```

Comment:
Let's put the cloud option first and start with that for workshops.

Reply:
Can!